sdk/videoanalyzer/video-analyzer-edge/samples/javascript/lvaInvokeModule.js

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+const { createRequest } = require("@azure/video-analyzer-edge");
```

Comment:
It would be good to add a good description about what the samples demonstrate and comments throughout the sample to help understand what's going on